docs/md/modules-covid.md

```diff
@@ -0,0 +1,707 @@
+                "num_samples" : "num_samples",
+                "vadr_outputs" : "vadr_outputs",
+                "cleaned_reads_unaligned_bams" : "cleaned_reads_unaligned_bams",
+                "demux_commonBarcodes" : "demux_commonBarcodes",
+                "submission_zip" : "submission_zip",
+                "cleaned_bams_tiny" : "cleaned_bams_tiny",
+                "data_tables_out" : "data_tables_out",
+                "ntc_rejected_batches" : "ntc_rejected_batches",
+                "picard_metrics_alignment" : "picard_metrics_alignment",
+                "failed_assembly_ids" : "failed_assembly_ids",
+                "ivar_trim_stats_html" : "ivar_trim_stats_html",
+                "assembly_stats_tsv" : "assembly_stats_tsv",
+                "failed_annotation_ids" : "failed_annotation_ids",
+                "run_date" : "run_date",
+                "genbank_source_table" : "genbank_source_table",
+                "num_read_files" : "num_read_files",
+                "ntc_rejected_lanes" : "ntc_rejected_lanes",
+                "primer_trimmed_read_count" : "primer_trimmed_read_count",
+                "gisaid_fasta" : "gisaid_fasta",
+                "num_submittable" : "num_submittable",
+                "submit_ready" : "submit_ready",
+                "passing_fasta" : "passing_fasta",
+                "nextclade_auspice_json" : "nextclade_auspice_json",
+                "read_counts_depleted" : "read_counts_depleted",
+                "cleaned_bam_uris" : "cleaned_bam_uris",
+                "num_assembled" : "num_assembled",
+                "max_ntc_bases" : "max_ntc_bases",
+                "genbank_fasta" : "genbank_fasta",
+                "multiqc_report_cleaned" : "multiqc_report_cleaned",
+                "num_failed_annotation" : "num_failed_annotation",
+                "meta_by_filename_json" : "meta_by_filename_json",
+                "primer_trimmed_read_percent" : "primer_trimmed_read_percent",
+                "assembly_stats_final_tsv" : "assembly_stats_final_tsv",
+                "demux_metrics" : "demux_metrics",
+                "submittable_ids" : "submittable_ids",
+                "sra_metadata" : "sra_metadata",
+                "spikein_counts" : "spikein_counts",
+                "raw_reads_unaligned_bams" : "raw_reads_unaligned_bams",
+                "ivar_trim_stats_tsv" : "ivar_trim_stats_tsv",
+                "picard_metrics_wgs" : "picard_metrics_wgs",
+                "nextclade_all_json" : "nextclade_all_json",
+                "multiqc_report_raw" : "multiqc_report_raw",
+                "sequencing_reports" : "sequencing_reports",
+                "demux_outlierBarcodes" : "demux_outlierBarcodes",
+                "nextmeta_tsv" : "nextmeta_tsv",
+                "assemblies_fasta" : "assemblies_fasta"
+            },
+            "identifier" : "run_id",
+            "name" : "Terra Workspace"
+            }
+        }
+    ```
+
+### Workload Model
```

Comment:
I don't know if the DB schema references here provide value in public-facing documentation.  If this section remains, could instead work better to represent this at a higher level.

General tip for sharing DB snippets publicly (Slack, JIRA, Confluence, etc), use expanded display:
```
wm111-e35:api okotsopo$ psql wfl
psql (11.11)
Type "help" for help.

wfl=# select * from workload limit 1;
 id |                  commit                  |            created            |                       creator                       | executor | finished | input | items | output | pipeline |             project             | release |           started           |                 uuid                 | version | wdl | stopped |      source_type      |          source_items           | executor_type | executor_items |     sink_type      | sink_items |            updated            |                         labels                          | watchers 
----+------------------------------------------+-------------------------------+-----------------------------------------------------+----------+----------+-------+-------+--------+----------+---------------------------------+---------+-----------------------------+--------------------------------------+---------+-----+---------+-----------------------+---------------------------------+---------------+----------------+--------------------+------------+-------------------------------+---------------------------------------------------------+----------
  1 | 73a852f36483818a9c89cc684448a360bc642fcd | 2021-06-15 10:35:12.801205-04 | wfl-non-prod@broad-gotc-dev.iam.gserviceaccount.com |          |          |       |       |        |          | (Test) okotsopo/GH-1326-sink-ns |         | 2021-06-15 10:35:12.9977-04 | a9ec9dd9-8885-4e0b-843b-54d99fe53cb0 | 0.8.0   |     |         | TDRSnapshotListSource | TDRSnapshotListSource_000000001 | TerraExecutor | 1              | TerraWorkspaceSink | 1          | 2021-06-15 10:45:08.706556-04 | {hornet:test,"project:(Test) okotsopo/GH-1326-sink-ns"} | {}
(1 row)

wfl=# \x
Expanded display is on.
wfl=# select * from workload limit 1;
-[ RECORD 1 ]--+--------------------------------------------------------
id             | 1
commit         | 73a852f36483818a9c89cc684448a360bc642fcd
created        | 2021-06-15 10:35:12.801205-04
creator        | wfl-non-prod@broad-gotc-dev.iam.gserviceaccount.com
executor       | 
finished       | 
input          | 
items          | 
output         | 
pipeline       | 
project        | (Test) okotsopo/GH-1326-sink-ns
release        | 
started        | 2021-06-15 10:35:12.9977-04
uuid           | a9ec9dd9-8885-4e0b-843b-54d99fe53cb0
version        | 0.8.0
wdl            | 
stopped        | 
source_type    | TDRSnapshotListSource
source_items   | TDRSnapshotListSource_000000001
executor_type  | TerraExecutor
executor_items | 1
sink_type      | TerraWorkspaceSink
sink_items     | 1
updated        | 2021-06-15 10:45:08.706556-04
labels         | {hornet:test,"project:(Test) okotsopo/GH-1326-sink-ns"}
watchers       | {}
```

Reply:
Thank you!

Reply:
I removed it. We can add later if necessary. I'd originally added it because some of the other modules contain similar information.